common/language.c

```diff
@@ -331,6 +331,16 @@ const char* const g_strings[StrId_Max][16] =
         STR_TW("应用"),
     },
 
+    [StrId_Actions_Star] =
+    {
+        STR_EN("Star"),
+    },
```

Comment:
For what it's worth, `STR_ES("Agregar a favoritos")`. This might be too long so maybe `STR_ES("Marcar")` would fit better. (Preferably long form should be used unless there are space constraints)

Reply:
The long form versions work fine, but take up more space leaving less for the path, let me know which you'd prefer.

![hbmenu_borrar](https://user-images.githubusercontent.com/7833953/60849539-172d3e80-a1e3-11e9-9339-797af7b40d71.jpg)
![hbmenu_desmarcar](https://user-images.githubusercontent.com/7833953/60849540-172d3e80-a1e3-11e9-8ee7-d6360d3e0751.jpg)

Reply:
Ah, the long form looks acceptable. However, what is that padding between the X button and the Minus button? I think it should be spaced in the same way as the other buttons. And I might be making this up, but the X button and text look vertically misaligned with respect to the other buttons, at least that's the visual impression I'm getting.

Reply:
Huh, vertically it's actually aligned with the A button, they're both slightly above the middle three. I'll check on that and the horizontal spacing.

Reply:
The A and X buttons were vertically aligned with the path, and better centred than the other three, so I've adjusted everything to match that. Fixed the spacing for the X label to match the others too.

Reply:
Also ES string added.